.changeset/wild-buses-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-defaults': patch
+---
+
+Added the option to skip database migrations by setting `skipMigrations: true` in options. This can be done globally in the database config or by plugin id.
```

Comment:
The one in options already existed, so this should just point out the new one in options right?